src/pages/Overview/OverviewPage.tsx

```diff
@@ -409,6 +414,34 @@ export class OverviewPage extends React.Component<OverviewProps, State> {
     );
   }
 
+  renderLabels(ns: NamespaceInfo): JSX.Element {
+    const button = (
+      <div style={{ color: ns.labels && PfColors.Blue400 }}>
+        {ns.labels ? Object.entries(ns.labels).length : 0} labels
```

Comment:
I think it's worth handling the "1 label" case.

Reply:
If there is 1 label so ns labels will be true and then Object.entries(ns.labels).length will be 1 .

I don't understand your proposal @jshaughn

Reply:
Sorry, it's just that I don't want to see "1 labels".  I'd prefer to add a test for the singular case and display "1 label". So,
0 labels
1 label
N labels

Reply:
Ohhh understand sorry I didn't see it